### DISCUSSION TITLE
How to run headless templates?

### DISCUSSION THREAD

**Discussion by xsser** [projectdiscovery/nuclei#1150] [Q&A] [answered]
**Describe the bug**
I've followed your advice to add -tags headless -headless to my command.But ,look at this
![图片](https://user-images.githubusercontent.com/1850771/137686215-870e3755-1d82-42df-a22f-43add2f25fe4.png)
why?
[FTL] Error, no templates were found.

**Nuclei version**
v2.3.8

**Comment by ehsandeep** [comment, marked as answer]
@xsser not sure which template you wanted to run? you don't need a headless flag or tag to run `CVE-2021-40438`, also `CVE-2021-40438.yaml` does not exist in your current directory, you can run it using the below cmd:

```
nuclei -target XX -t cves/2021/CVE-2021-40438.yaml
```

Also make sure you are running the latest version of the project, i.e [v2.5.2](https://github.com/projectdiscovery/nuclei/releases/tag/v2.5.2)

```
go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
```
